New package: OptimizationManopt v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: OptimizationManopt
- Repository: https://github.com/SciML/Optimization.jl
- Created by: @Vaibhavdixit02
- Version: v0.0.1
- Commit: cedf962ce34b4298e4262d8ab225e79b4de8faba
- Reviewed by: @Vaibhavdixit02
- Reference: https://github.com/SciML/Optimization.jl/issues/16#issuecomment-2147630652
- Description: Mathematical Optimization in Julia. Local, global, gradient-based and derivative-free. Linear, Quadratic, Convex, Mixed-Integer, and Nonlinear Optimization in one simple, fast, and differentiable interface.
<!-- 45a4cd22be5b6a15fc3c6d377732d7af199962d02b6bc24b247853703bb43997284333f09484ecc44bb3c4ee9b5056ab61fad8cc4c65be38de6e87734d3c4d5b533f99830705b4457b91649a6e011c0b6ae53c9cbdb86b3d19a7e6040b12f71cfe0d2db4dc1aa84d4a09cfddb83f2261323c038b89c78569e9af85b4e51bf232ea7b8c7e0907aa5c86f06bcab6688c36467869556caa7962b6edce43c37a644964a2b9b98a1c236b9dd80a06c3b07d80ee71489d4922cae3d048f1ee76546e7e -->

goerz commented:
> No licenses detected in the package's top-level folder. An OSI-approved license is required.

That's a little weird. Not sure why it doesn't find the license.

Other than that, looks good. Don't forget to update the [section in the README](https://github.com/SciML/Optimization.jl?tab=readme-ov-file#installation) to point to the new package.

[noblock]

Vaibhavdixit02 commented:
[noblock] I guess it needs the LICENSE in the subdir, I'll add it. Thanks!

JuliaRegistrator committed:
New package: OptimizationManopt v0.0.1

UUID: e57b7fff-7ee7-4550-b4f0-90e9476e9fb6
Repo: https://github.com/SciML/Optimization.jl.git
Tree: 37b0e62a110ba2133147bbfb35be446c63772edd

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

GunnarFarneback commented:
[noblock]

Yes, only the subdir is distributed by the package servers, so there needs to be a license inside.
